test/e2e/common_test.go

```diff
@@ -106,12 +106,15 @@ var (
 	tempdir    string
 	err        error
 	podmanTest *PodmanTestIntegration
+	safeIP     int
 
 	_ = BeforeEach(func() {
 		tempdir, err = CreateTempDirInTempDir()
 		Expect(err).ToNot(HaveOccurred())
 		podmanTest = PodmanTestCreate(tempdir)
 		podmanTest.Setup()
+		// see GetSafeIPAddress() below
+		safeIP = (GinkgoT().ParallelProcess()+128)*256 + 2
```

Comment:
maybe choose a different name, this is not a ip, it is a single integer.

Reply:
Good point, is `safeIPOctets` more sensible?

Reply:
yes